### ISSUE TITLE
Getid system

### ISSUE THREAD

**Issue by Sqeed** [Sqeed/ModBotMkII#10]
ModBotMkI had `+getid <name>` command which worked the following way:

ModBot would monitor gamertags channel and if a messages was received which contained something that resembled a battle.net tag it saved it in a dictionary with the user ulong Id being the key and the battle.net tag being the value. The command would then TryGet from that library and represent the stored value. It would then use AngeSharp to determine region and competitive rank.

**Comment by Sqeed** [comment]
Opened because this is a core feature of the old bot. We should make this customizable with server settings (like which text channel to survey). We should analyse the message the bot recieves and check for common gaming system IDs, and then try to save them in server settings, so that the getid command returns all common gaming platform IDs and not just battle.net and overwatch info. Also make a command to take the battle.net ID from that collection and get player stats.



Tldr: getid command gives user gaming platform IDs, while new command gives overwatch player stats.
